docs/getting-started/croeseid-testnet-nix.md

```diff
@@ -20,7 +20,7 @@ $ cachix use crypto-com
 ## Install chain utils for testnet
 
 ```shell
-$ nix-env -iA chain-utils-testnet -f https://github.com/crypto-com/chain-main/archive/95674386b63f6dba1aa4a5c6f16e8755be8408bc.tar.gz
+$ nix-env -iA chain-utils-testnet -f https://github.com/crypto-org-chain/chain-main/archive/v1.1.0.tar.gz
```

Comment:
this should be mainnet source code.
Should it be https://github.com/crypto-org-chain/chain-main/archive/v0.9.1-croeseid.tar.gz

Reply:
good catch, fixed.